go.mod

```diff
@@ -36,6 +36,7 @@ require (
 	github.com/golang/protobuf v1.5.2 // indirect
 	github.com/golang/snappy v0.0.4 // indirect
 	github.com/klauspost/compress v1.12.3 // indirect
+	github.com/kr/pretty v0.2.1 // indirect
```

Comment:
wait this is not eth?

Reply:
not, but I see this add into here when run  go get github.com/ethereum/go-ethereum

Reply:
also the latest is `v1.10.21` I think the problem is latest has some issues. check out line 12 - we use latest. so we can't do anything for now